src/System.Windows.Forms/tests/AccessibilityTests/Student.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AccessibilityTests
+{
+    public class Student
+    {
+        public Student(int studentNo, string studentName, string studentSex)
+        {
+            StudentNo = studentNo;
+            StudentName = studentName;
+            StudentSex = studentSex;
+        }
+
+        public Student(int studentNo, string studentName, string studentSex, int studentPhoneNum, string homeNumber, string student_habit, bool is_Student, int student_Count, int Lucky_number)
```

Comment:
Incorrect naming:
```suggestion
        public Student(int studentNumber, string studentName, string studentSex, int studentPhoneNumber, string homeNumber, string studentHabit, bool isStudent, int studentCount, int luckyNumber)
```

* Should it be `studenHomeNumber`? And what's the difference with `studentPhoneNumber`?
* Should it also be `studentLuckyNumber`?
* Why do we need `isStudent` flag? An instance of "Student" with `isStudent = false` is a very strange thing...
* What's `studentCount`?

Reply:
Is there a need for "student" prefix at all since we're in a context of a "Student" object?
```suggestion
        public Student(int studentNumber, string name, string sex, int phoneNumber, string homeNumber, string hobby, bool isStudent, int student_Count, int luckyNumber)
```

The same Q applies to the type properties.

Reply:
studentCount & isStudent just for testing, don't worry about it.